File: game-server/src/main/java/work/fking/pangya/game/packet/handler/MyRoomOpenPacketHandler.java
package work.fking.pangya.game.packet.handler;

import io.netty.buffer.ByteBuf;
import work.fking.pangya.game.net.ClientPacketHandler;
import work.fking.pangya.game.GameServer;
import work.fking.pangya.game.player.Player;

public class MyRoomOpenPacketHandler implements ClientPacketHandler {

    @Override
    public void handle(GameServer server, Player player, ByteBuf packet) {
        var selfUserId = packet.readIntLE();
        var targetUserId = packet.readIntLE();
    }
}

File: game-server/src/main/kotlin/work/fking/pangya/game/packet/handler/myroom/MyRoomOpenPacketHandler.kt
package work.fking.pangya.game.packet.handler.myroom

import io.netty.buffer.ByteBuf
import work.fking.pangya.game.GameServer
import work.fking.pangya.game.net.ClientPacketHandler
import work.fking.pangya.game.player.Player

class MyRoomOpenPacketHandler : ClientPacketHandler {
    override fun handle(server: GameServer, player: Player, packet: ByteBuf) {
        val selfPlayerUid = packet.readIntLE()
        val targetPlayerUid = packet.readIntLE()
    }
}
